container-terminal.js

```diff
@@ -105,6 +105,9 @@
                         var defaultCols = 80;
                         var defaultRows = 24;
 
+                        /* Whether the component is preventing a connection */
+                        var preventing = true;
```

Comment:
The change looks fine, but the var name is a bit confusing (particularly because we have a scope var called `prevent`). Can we call this something like `connectionRequested` or `overridePrevent` and flip the logic? I think it would make the intent more clear.